### ISSUE TITLE
Argument to --timeout needs to be positiv

### ISSUE THREAD

**Issue by NobbZ** [oleander/git-fame-rb#53]
# Reproduce

```plain
linux-kernel$ git fame
Error: Maximum timeout of 10 sec exceed. Try passing --timeout=<sec> or -1 for no timeout.
Try --help for help.

linux-kernel$ git fame --help
git-fame 2.2.0 (c) 2012-2016 Linus Oleander
[...]
  -m, --timeout=<i>         Set timeout in seconds for each git command. Turn off by using -1 (default: 10)
[...]

linux-kernel$ git-fame --timeout=-1
Error: argument --timeout must be positive.
Try --help for help.
```

# Expected behaviour

Do not error and do not timeout when called with `--timeout=-1` or do not mention `-1` for no timeout in help at all, but I'd really prefer to have an option to put off time outs.

**Comment by oleander** [comment]
Hi! Could you run the first cmd with `--verbose` and post the output?

**Comment by oleander** [comment]
Just released 2.3. Could you try it?

``` bash
gem cleanup git_fame
gem install git_fame --version 2.3.0
git fame
```

**Comment by NobbZ** [comment]
It does now run without timing out, but crashing due to OOM ;) So I do think the linux source history is simply to large for a full scan ;)
